src/cli/cli_coap.cpp

```diff
@@ -328,15 +435,117 @@ template <> otError Coap::Process<Cmd("parameters")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli coap get
+ * @code
+ * coap get fdde:ad00:beef:0:2780:9423:166c:1aac test-resource
+ * Done
+ * @endcode
+ * @code
+ * coap get fdde:ad00:beef:0:2780:9423:166c:1aac test-resource block-1024
+ * Done
+ * @endcode
+ * @cparam coap get @ca{address} @ca{uri-path} [@ca{type}]
+ *   * `address`: IPv6 address of the CoAP server.
+ *   * `uri-path`: URI path of the resource.
+ *   * `type`:
+ *         * `con` for Confirmable or `non-con` for Non-confirmable (default).
+ *         * `non-con`: Non-confirmable (default)
```

Comment:
non-con is both in the first and second bullet

This happens in some other commands, please update those as well to align with what you do here.